2019/pt-br/src/0xb0-next-devs.md

```diff
@@ -0,0 +1,31 @@
+Próximos passos para Desenvolvedores
+====================================
+
+A tarefa de criar e manter software seguro, ou, corrigir software existente, pode ser uma tarefa difícil. APIs não são diferentes.
+
+Nós acreditamos que a educação e conscientização são fatores chave para escrever software seguro. Tudo o que é necessário para alcançar este objetivo depende de **estabelecer e usar repetidamente processos de  e padronizar controles de segurança**.
+
+O OWASP possui um grande número de fontes abertas para endereçar a segurança desde o princípio dos projetos. Por favor, visite a [página de projetos do OWASP][1] para uma extensa lista de projetos disponíveis.
+
+| | |
+|-|-|
+| **Educação** | Você pode iniciar lendo [os materiais de educação do OWASP][2] de acordo com sua área ou interesse. Para aprendizado mão-na-massa, nós adicionarmos **crAPI** - **C**ompletely **R**idiculous **API** (API Ridiculamente Vulnerável) em [nosso roadmap][3]. Enquanto isso, você pode praticar segurança de aplicações web utilizando o [OWASP DevSlop Pixi Module][4], um aplicativo web vulnerável e um serviço API com a intenção de ensinar usuário como testar aplicações web modernas e também APIs com relação a problemas de segurança. Você pode também participar das [conferências AppSec do OWASP][5] com sessões de treinamento, ou ainda [juntar-se a seu capítulo local][6]. |
+| **Requisitos de Segurança** | Segurança deve fazer parte de qualquer projeto desde o princípio. Ao fazer a escolha de requisitos, é importante definir o que a "segurança" representa para o projeto. O OWASP recomenda o uso do [OWASP Application Security Verification Standard (ASVS)][7] como um guia para atribuir requisitos de segurança. Se você trabalha com *outsourcing*, considere o projeto [OWASP Secure Software Contract Annex][8], o qual deve ser adaptado de acordo com a leis e regulamentos locais. |
```

Comment:
fix
* where it's written "_(...) o qual deve ser adaptado de acordo com **a** leis e regulamentos locais._"
* it should be "_(...) o qual deve ser adaptado de acordo com **as** leis e regulamentos locais._"

Reply:
okay